src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheckExamplesTest.java

```diff
@@ -34,36 +33,36 @@ protected String getPackageLocation() {
     @Test
     public void testExample1() throws Exception {
         final String[] expected = {
-
+                getCheckMessage(MSG_KEY),
         };
 
-        verifyWithInlineConfigParser(getPath("Example1.txt"), expected);
+        verifyWithInlineConfigParser(getPath("Example1.java"), expected);
     }
 
     @Test
     public void testExample2() throws Exception {
         final String[] expected = {
-
+                getCheckMessage(MSG_KEY)
         };
 
-        verifyWithInlineConfigParser(getPath("Example2.txt"), expected);
+        verifyWithInlineConfigParser(getPath("Example2.java"), expected);
     }
 
     @Test
     public void testExample3() throws Exception {
         final String[] expected = {
-
+                getCheckMessage(MSG_KEY)
         };
 
-        verifyWithInlineConfigParser(getPath("Example3.txt"), expected);
+        verifyWithInlineConfigParser(getPath("Example3.java"), expected);
     }
 
     @Test
     public void testExample4() throws Exception {
         final String[] expected = {
-
+                getCheckMessage(MSG_KEY)
```

Comment:
You haven't told the parser about which violations to ignore.. so, it is throwing all of them back at you :)

Reply:
Hello Manish, Sorry for delayed reply. The problem is that I do not know where to get violation types to be ignored. I have seen PRs [1](https://github.com/checkstyle/checkstyle/commit/7123cc2b7ca95f5f349e54776a02724f2e7b0d69#diff-8b9ed11f4d5073c3d06414a4a2e976ff103dfad4e90cac65009cfa7648cb28b2R38), [2](https://github.com/checkstyle/checkstyle/commit/ddbcf447682a58a9de237391dad99e96903cb969?diff=split&w=0), [3](https://github.com/checkstyle/checkstyle/commit/2cec4d51ffb0478689039251c66a4e0253d6ec9f?diff=split&w=0) But I didn't understand what are "xx:xx" numbers they concatenate with `                getCheckMessage(MSG_KEY)`.

Reply:
My wording in previous msg wasn't clear.. We do not ignore violations, the input files are made weird by design to test. You need to tell the parser where to expect a violation so that it bypasses from throwing a failure/error and this way we know where we are inducing a violation and that is accurately being logged.
As for getting this PR through finish line, the "xx:xx" is the line:column where the violation occurs.

Reply:
@MANISH-K-07 

I have tried all the combinations but to no avail. Unfortunately, I do not know where to read about the violations or how to do it rather than just imitating.

All the following  tries were committed (but not pushed).
```java
// Try 1
getCheckMessage(MSG_KEY, "13, 15, 40"),

// Try 2
"13:30" + getCheckMessage(MSG_KEY),
"15:30" + getCheckMessage(MSG_KEY),
"40:12" + getCheckMessage(MSG_KEY),

// Try 3
getCheckMessage(MSG_KEY, 13, 15, 40),
```

Sorry If I missed something (documentation or a trial) but I the debugging couldn't help me to figure out what are the expected exactly (rather than just some lines that throws violations)